inc/composer/class-command.php

```diff
@@ -116,6 +120,40 @@ private function get_base_command_prefix() : string {
 		);
 	}
 
+	/**
+	 * Get all extra configs added by all the packages.
+	 *
+	 * @return array Extra configurations added by packages (e.g. handlers).
+	 */
+	protected function get_package_configs() : array {
+		// Fetch all the packages.
+		$repo = $this->getApplication()->getComposer()->getLocker()->getLockedRepository( true );
+		$packages = [];
+		foreach ( $repo->getPackages() as $package ) {
+			$packages[ $package->getName() ] = $package;
+		}
+
+		// Check which are publishing containers.
+		$config = [];
+		foreach ( $packages as $name => $package ) {
+			$extra = $package->getExtra();
+			if ( ! isset( $extra['altis'] ) || ! isset( $extra['altis']['local-server'] ) ) {
```

Comment:
This would be clearer as a positive test:
```suggestion
			if ( isset( $extra['altis'] ) && isset( $extra['altis']['local-server'] ) ) {
				$config[ $name ] = $extra['altis']['local-server'];
			}
```

Reply:
I'd agree if it were a straight `if`/`else`, but this is more of a return-early pattern. Not strongly opinionated so if you feel it's much more readable that way I'll update

Reply:
Maybe `empty` would read better. `if (empty( $extra['altis']) || (empty($extra['altis']['local-server']))`